content/blog/blog-post-06.md

```diff
@@ -0,0 +1,211 @@
+---
+authors: ["kokkos-team"]
+title: "17 Apr. 2024: First Kokkos tea-time"
```

Comment:
Do you intend to post a new page each month?

Reply:
I was thinking to only make this one, otherwise the blog section will be invaded by this. As you say in another comment, making a dedicated section could be an option though

Reply:
Then maybe drop the date from the title and just say we are launching the series.
Also in that case you might not want to have the title to the 1st presentation in the picture.

Reply:
Let me know what you feel about the new version, would you prefer I completely remove even the mention of the first instance?